commons/src/methods/convertDateTimeZone.ts

```diff
@@ -0,0 +1,43 @@
+import { DateTime } from "luxon";
+
+/**
+ *
+ * @param slot Date
+ * @param zone string
+ * @returns Date | string
+ */
+export function formatTimeSlot(slot: Date, zone: string): Date | string {
```

Comment:
It feels a bit weird to return either `Date | string` here. In the cases where we're returning the original `slot`, can we return an empty string instead?

Reply:
Maybe I'm missing something here, but wouldn't that cause the UI to not show any ticks at all?